Locales/enUS.lua

```diff
@@ -102,9 +102,9 @@ AtrL["Show auction house prices in mailbox tooltips"] = ""
 AtrL["Undercutting"] = ""
 AtrL["Selling"] = ""
 AtrL["Configure how you typically like to sell the items listed below."] = ""
-AtrL["Clears"] = "Pulizia"
+AtrL["Clears"] = ""

```

Comment:
I think this should be "Cleanup" or "Clear Data"
not an empty string

Reply:
Done